main.py

```diff
@@ -47,6 +48,7 @@
 supported_currencies = get_all_currencies(single=True)
 logger.debug("Finished get_all_currencies")
 
+send_telegram("new-coin-bot is online")
```

Comment:
As discussed in Discord (starting around here: https://discord.com/channels/900656563805642753/900658475489714176/911437884156952597) the whole Telegram thing needs to be opt-in. Another configuration variable would be necessary that defaults to `False`.

Reply:
will do